plugins/renepay/pay.c

```diff
@@ -624,14 +680,17 @@ payment_listsendpays_previous(
 
 	json_for_each_arr(i, t, arr)
 	{
-		u64 partid, groupid;
+		u64 partid = 0, groupid;
 		struct amount_msat this_msat, this_sent;
 		const char *status;
 
-		// TODO(eduardo): assuming amount_msat is always known.
+		// TODO: we assume amount_msat is always present, but according
+		// to the documentation this field is optional. How do I
+		// interpret if amount_msat is missing?
```

Comment:
amount_msat is never missing on payments renepay makes.  Older payments, sure...